### ISSUE TITLE
Crashes when using moz-jpeg plugin

### ISSUE THREAD

**Issue by ghost** [ahmadnassri/node-metalsmith-imagemin#2]
When trying to use metalsmith-imagemin with [my project](https://github.com/ismay/ismaywolff.nl/tree/bca925702ebdc92e302200ca60c984802422c0e4) and the moz-jpeg plugin, I get the following error:

```
╰─$ make build
node build.js
/home/ismay/Github/ismaywolff.nl/build.js:120
    if (err) throw err;
                   ^
TypeError: Cannot read property 'bind' of undefined
    at minify (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-imagemin/src/index.js:25:37)
    at /home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-imagemin/node_modules/async/lib/async.js:122:13
    at _each (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-imagemin/node_modules/async/lib/async.js:46:13)
    at Object.async.each (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-imagemin/node_modules/async/lib/async.js:121:9)
    at Ware.<anonymous> (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-imagemin/src/index.js:62:11)
    at Ware.<anonymous> (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith/node_modules/ware/node_modules/wrap-fn/index.js:45:19)
    at next (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith/node_modules/ware/lib/index.js:85:20)
    at /home/ismay/Github/ismaywolff.nl/node_modules/metalsmith/node_modules/ware/node_modules/wrap-fn/index.js:121:18
    at /home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-assets/index.js:50:9
    at done (/home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-assets/node_modules/async/lib/async.js:128:19)
    at /home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-assets/node_modules/async/lib/async.js:25:16
    at /home/ismay/Github/ismaywolff.nl/node_modules/metalsmith-assets/index.js:66:11
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:382:3)
make: *** [build] Error 1

```

But I'm using the example from your readme. Is this is a bug, or am I doing something wrong?

**Comment by ahmadnassri** [comment]
I think my example was not clear, first, you don't actually need `mozjpeg` to run `imagemin` on `jpeg` files, it actually comes with all standard processors, so unless you specifically want to run `mozjpeg` it should just work out of the box.

second, I fixed the issue now, and you should be able to use any plugin properly :)

released in `v1.1.2`

**Comment by ghost** [comment]
Cool, can confirm imagemin-mozjpeg works with `1.1.2`. Wasn't confused by the readme example by the way. I understood you don't need it. I just needed a higher compression ratio. I'd say the example use of a plugin was very helpful!

Thanks for the quick response by the way!
